internal/provider/models/destinations/http.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
 	"github.com/hashicorp/terraform-plugin-framework/resource/schema/stringdefault"
 	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+	"github.com/hashicorp/terraform-plugin-framework/types/basetypes"
```

Comment:
This looks like a dupe, but not exported to `.`. It should be one or the other.

Reply:
Yes, I'd stick to one import style or the other. Using both is a bit confusing.